src/main/java/org/cascadebot/cascadebot/commands/moderation/BanCommand.java

```diff
@@ -42,14 +43,14 @@ public void onCommand(Member sender, CommandContext context) {
         if (targetMember == null) {
             // If the member is null, the user does not exist in the guild.
             // This attempts to retrieve the user from Discord.
-            targetUser = DiscordUtils.getUser(context.getGuild(), context.getMessage(0), true);
-
-            if (targetUser == null) {
+            try {
+                targetUser = DiscordUtils.getUser(context.getGuild(), context.getMessage(0), true);
+            } catch (ErrorResponseException | NumberFormatException e) {
```

Comment:
We should really be editing DiscordUtils to handle this

Reply:
Yea weeryan said that in the last PR that I did, but you said for beta. Are we doing that here now?